docusaurus/docs/publish-a-plugin/sign-a-plugin.md

```diff
@@ -15,15 +15,30 @@ keywords:
 
 All Grafana Labs-authored backend plugins, including Enterprise plugins, are signed so that we can verify their authenticity with [signature verification](https://grafana.com/docs/grafana/latest/administration/plugin-management#plugin-signatures). By [default](https://grafana.com/docs/grafana/latest/administration/plugin-management#allow-unsigned-plugins), Grafana requires all plugins to be signed in order for them to be loaded.
 
-Before you can sign your plugin, you need to decide whether you want to sign it as a _public_ or a _private_ plugin.
+:::info
+
+It's not necessary to sign a plugin during development. The [Docker development environment](../get-started/set-up-development-environment.mdx) that is scaffolded with `@grafana/create-plugin` will load the plugin without a signature. This is because it is configured by default to run in [development mode](https://github.com/grafana/grafana/blob/main/contribute/developer-guide.md#configure-grafana-for-development).
+
+:::
+
+## Public or private plugins
+
+Plugins can have different [signature levels](https://grafana.com/legal/plugins/#what-are-the-different-classifications-of-plugins) according to their author, related technology, and intended use.
+
+The plugin can be _public_, signed as Community or Commercial, and distributed within the [Grafana plugin catalog](https://grafana.com/plugins), so that it is available for others to install. Or you can make it _private_ and only available for use within your organization.
+
+Before signing your plugin, review the [Plugins policy](https://grafana.com/legal/plugins/) to determine the appropriate signature for your plugin.
+
+## Signature levels
```

Comment:
Commenting again to place it ^ sig levels ection

Reply:
It is above where the sig levels section used to be.